src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Event} from 'app/types';
+import {Metadata} from 'app/sentryTypes';
+import {getTitle} from 'app/utils/events';
+
+type Props = {
+  data: {
+    type:
+      | 'error'
+      | 'csp'
+      | 'hpkp'
+      | 'expectct'
+      | 'expectstaple'
+      | 'default'
+      | 'transaction';
```

Comment:
Should this union be in app/types? It feels like something that will be used in more than one place and could be part of the event type too.